android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java

```diff
@@ -143,6 +141,10 @@ private void loadData() {
 
             sessions.clear();
             sessions.addAll(track.getSessions().sort("startsAt"));
+            sessionsListAdapter.setCopyOfSessions(track.getSessions().sort("startsAt"));
+            if (!Utils.isEmpty(searchText))
```

Comment:
What happens if a user types something and clears it?

Reply:
It will work fine. If user types something onQueryTextChange(String query) will be called and list will be filtered.

loadData() method is called when fetching data from Realm. This  
```
if (!Utils.isEmpty(searchText))
   sessionsListAdapter.filter(searchText);
``` 
will not filter the list on first time and will decrease loading time